app/src/main/res/values-pt/strings.xml

```diff
@@ -52,7 +52,7 @@
     <string name="icon_designer">Designer do Ícone (Famil Qasimov)</string>
 
     <string name="less_than_two">Erro! Não dá para calcular a diferença com menos de dois items.</string>
-    <string name="empty_please_select_two">Selecione dois items abaixo\npara comparar as diferenças</string>
+    <string name="empty_please_select_two">Selecione dois itens abaixo\para comparar as diferenças</string>
```

Comment:
The "\npara" is acually to break the line (\n) before the word "para", there is nothing wrong on it.

Reply:
Ah ok sorry

Reply:
In a "best case scenario" this breakline shouldn't be needed and be auto, but the cardview looks a lot better with it :stuck_out_tongue:

Reply:
That's okay 😊